WalkingChallenge/ProfileViewController.swift

```diff
@@ -242,26 +243,26 @@ class ProfileViewController: UIViewController {
   }
 
   private func configureHeaderView(_ top: inout ConstraintRelatableTarget) {
-    view.addSubview(profileImage)
+    scrollView.contentView.addSubview(profileImage)
     profileImage.contentMode = .scaleAspectFill
     // TODO(compnerd) figure out how to get this value properly
     profileImage.layer.cornerRadius = Style.Size.s128 / 2.0
     profileImage.layer.masksToBounds = true
     profileImage.snp.makeConstraints { (make) in
-      make.top.equalTo(top).offset(Style.Padding.p12)
+      make.top.equalToSuperview().offset(Style.Padding.p12)
```

Comment:
Changing the initialization of the parameter below should prevent this bit of the change.